### ISSUE TITLE
Not working on Google

### ISSUE THREAD

**Issue by wapkanishka** [ajartille/gumshoe#16]
With new google interface this extension not record data in google accounts

**Comment by ajartille** [comment]
Yes, there are increasingly many website which do not use standard text/password fields for entry, and Google is among them... In the next update, I will try to address these shortcomings. I've been keeping note of websites which don't work anymore as I notice them. I'll close this ticket when it's fixed. Thanks!
